include/DataFrame/Utils/Matrix.h

```diff
@@ -95,8 +95,16 @@ class   Matrix  {
     reference operator() (size_type r, size_type c);
     const_reference operator() (size_type r, size_type c) const;
 
-    reference operator[] (size_type r, size_type c);
+#if !defined(__cpp_multidimensional_subscript) || (defined(_MSC_VER) && _MSC_VER <= 1944 /* MSVC2022 v17.14.0 */ )
+#if 0
+#pragma message("Your compiler does not fully support C++23: operator[] with multiple arguments is not properly supported as per https://en.cppreference.com/w/cpp/language/operators / https://isocpp.org/wiki/faq/operator-overloading#matrix-subscript-op / https://en.cppreference.com/w/cpp/feature_test#Language_features")
+#endif
```

Comment:
What is the purpose of these lines?
Thx

Reply:
Initially I wrote this to get the compiler to report an informative message at compile time (to help anyone understand *why* it behaves the way it does) but it turned out to be too noisy, cluttering my builds, so it got disabled.

(Todo
@me: remove)